tests/integration/shared/sources/new-or-changed-sources-test.js

```diff
@@ -72,12 +68,24 @@ async function runCrawl (key, today) {
   return result
 }
 
+/** Gets the dates for a given source key.
+ *
+ * For example, if the cache only contained crawler-cache/nyt/2020-04-01,
+ * this would return ['2020-04-01'].
+ */
+function getCacheDatesForSourceKey (key) {
+  const cacheRoot = join(__dirname, '..', '..', '..', '..', 'crawler-cache')
+  const folders = glob.sync(join(cacheRoot, key, '*/'))
+  return folders.map(f => f.split(sep)).
```

Comment:
`f.split(sep)[1]`

Reply:
Fixed.  split(sep)[1] wasn't sufficient, but a regex replace was, and no nested maps now.